habitat-lab/habitat/utils/visualizations/utils.py

```diff
@@ -291,3 +289,76 @@ def append_text_to_image(image: np.ndarray, text: str):
     text_image = blank_image[0 : y + 10, 0:w]
     final = np.concatenate((image, text_image), axis=0)
     return final
+
+
+def append_text_to_image(
+    image: np.ndarray, text: List[str], font_size: float = 0.5
+):
+    r"""Appends lines of text on top of an image. First this will render to the
+    left-hand side of the image, once that column is full, it will render to
+    the right hand-side of the image.
+    :param image: the image to put text underneath
+    :param text: The list of strings which will be rendered, separated by new lines.
+    :returns: A new image with text inserted underneath the input image
```

Comment:
I think this comment line is wrong.

Reply:
This function was moved here to be in visualisations. The implementation is the same as before (I think it is a result of copy paste). I can correct that.